docs/notebooks/scatterwidget.ipynb

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
<p>looks good, but can you actually show/print in the notebook what it looks like? and also maybe do 1 visualization (e.g. plot cell types etc)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/napari-spatialdata/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>